pulp_container/app/registry_api.py

```diff
@@ -962,6 +970,9 @@ def get(self, request, path, pk):
     @RegistryApiCache(base_key=lambda req, cac: find_base_path_cached(req, cac))
     def handle_safe_method(self, request, path, pk):
         """Handles safe requests for Blobs."""
+        # Fail early in case of unsupported manifest schemas
+        self.handle_old_manifest_version(request.accepted_media_type, pk)
```

Comment:
We do not care about media types when it comes to blobs. You can safely remove this check here.